cpp/ql/lib/change-notes/2023-04-28-static-local-dataflow.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The new dataflow (`semmle.code.cpp.dataflow.new.DataFlow`) and taint-tracking libraries (`semmle.code.cpp.dataflow.new.TaintTracking`) now supports tracking flow through static local variables.
```

Comment:
```suggestion
* The new dataflow (`semmle.code.cpp.dataflow.new.DataFlow`) and taint-tracking libraries (`semmle.code.cpp.dataflow.new.TaintTracking`) now support tracking flow through static local variables.
```

Reply:
Oh, sorry. It was auto-resolved after I accepted the suggestion 😄. You're right: We did support dynamically initialized static variables before. I'll update the change note to reflect this.

Reply:
So what I'm asking about here is incorrect. See also the whole discussion above. So leave as-is, please.